### PR TITLE
Ansible: custom URL for k8s binaries

### PR DESCRIPTION
**Pull request by papagalu**
fixes: #495

### CODE REVIEW

File: contrib/roles/linux/kubernetes/tasks/download_k8s_binaries.yml [hunk 1]
```diff
@@ -44,6 +44,8 @@
         remote_src: yes
 
     - name: Kubernetes bins | download kubernetes binaries
+      environment:
+        http_proxy: "{{KUBERNETES_DOWNLOAD_PREFIX}}"
```
Comment (alinbalutoiu) [inline contrib/roles/linux/kubernetes/tasks/download_k8s_binaries.yml]: Is this working? You're trying to download the binaries using the proxy "https://dl.k8s.io"?

Comment (papagalu) [reply contrib/roles/linux/kubernetes/tasks/download_k8s_binaries.yml]: nope, i messed up the env var, sorry for that, i fixed it

**Comment by adelina-t** [comment]
@alinbalutoiu Basically the ideea is to use this in a CI env and use prebuilt binaries from master. But I don't think this is the right approach. As you said, we can just edit the variable files to add whatever custom URL.

**Comment by papagalu** [comment]
This no more needed. Problem is fixed in #535

**Pull request closed without merging**